### DISCUSSION TITLE
Microsoft Teams notification window (bottom right) not appearing on remote system since early last week

### DISCUSSION THREAD

**Discussion by kingmustard** [rustdesk/rustdesk#7470] [General]
### Bug Description

The local system runs Windows 10 x64 and Microsoft Teams v24046.2809.2757.3796.

When it receives a message or call, the notification appears on the screen (bottom right) correctly.

However, as of early last week, the notification window is not visible when I remotely connect into the laptop via RustDesk.

Notifications settings are working perfectly on the local system itself. They are set correctly within Teams and Windows Settings.

Tested with TeamViewer and AnyDesk, everything works correctly with those.

### How to Reproduce

When Teams receives a message or call, the notification box appears on the local screen (bottom right) as expected.

The notification is visually invisible when I remotely connected into the laptop, though can hear the ringtone etc

### Expected Behavior

Notification box visually appears on both local and remote systems.

### Operating system(s) on local side and remote side

Windows 10 x64 local, Windows 11 x64 remote

### RustDesk Version(s) on local side and remote side

1.2.3 local, 1.2.3 remote

### Screenshots

Not applicable.

**Comment by kingmustard** [comment]
Hi there,

Is there any update on this, please?

**Comment by Flygro** [comment]
Also happening for me on Windows 11 22H2, Teams 24033.813.2773.520

**Comment by JMort92** [comment]
Also experiencing this, on the "New" Teams.

A short term workaround (for me at least) is to revert to Classic Teams, which I understand uses a different type of notification. However, Classic is being phased out.

**Comment by Paddy-M123** [comment]
@kingmustard - we are experiencing the same behavior since we are using Teams 2.0
We tested with RealVNC and TeamViewer - both don't show the Popup remotely, although it is present on the screen.
Also we can reproduce it when we are in a Teams Session with Screen sharing.

Any news from your end?
Did you - by chance - open a ticket at Microsoft?

**Reply by kingmustard** [reply]
No news on my end, unfortunately.

I did not raise a ticket with Microsoft as I imagine they will either:

1. respond generically i.e. please try reinstall Teams, make sure you have all Windows updates installed etc
2. say it is a problem with RustDesk, Teamviewer etc

This has been an issue for a very long time. Not sure if the issue is with:

1. Teams itself (both original and 'New') and their potential use of non-standard Windows notifications (I think this is most likely)
2. the way Windows handles certain kinds of notifications
